package.json

```diff
@@ -4,13 +4,21 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint ."
```

Comment:
lets change script to : eslint --fix . 
this will automatically fix most issues when we call it , for example it will replace all double quote with single quote